app/controllers/conferences_controller.rb

```diff
@@ -48,7 +55,7 @@ def conference_params
   end
 
   def confirm_role
-    redirect unless current_user.admin? || current_user.student?
+    redirect_to root_path, alert: 'You are not authorized to access this page.' unless current_user.admin? || current_user.current_student?
```

Comment:
This is somehow duplicating what you define as abilities to `crud` conferences - could you also reuse the ability here in the controller?